src/enigma2/Admin.h

```diff
@@ -65,6 +67,7 @@ namespace enigma2
     utilities::StreamStatus GetStreamDetails(const std::shared_ptr<data::Channel> &channel);
     void GetTunerDetails(utilities::SignalStatus &signalStatus, const std::shared_ptr<data::Channel> &channel);
 
+    std::string m_addonVersion;
```

Comment:
As you only assign a value once in the ctor of the class, this member can be const.

Reply:
Changed